libraries/lib-stretching-sequence/ClipInterface.cpp

```diff
@@ -0,0 +1,3 @@
+#include "ClipInterface.h"
+
+ClipInterface::~ClipInterface() = default;
```

Comment:
You did make a tiny translation unit for this one

Reply:
Because I initially added this in `lib-mixer` and wanted to be consistent with the code around me :D